openmetadata-docs/content/v1.4.x/connectors/database/teradata/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Teradata
+slug: /connectors/database/teradata
+---
+
+{% connectorDetailsHeader
+name="Teradata"
+stage="BETA"
+platform="OpenMetadata"
+availableFeatures=["Metadata", "Data Profiler"]
+unavailableFeatures=["Query Usage", "Data Quality", "Owners", "Tags", "Stored Procedures", "Lineage", "Column-level Lineage", "dbt"]
+/ %}
+
+In this section, we provide guides and references to use the Teradata connector.
+
+Configure and schedule Teradata metadata and profiler workflows from the OpenMetadata UI:
+
+- [Requirements](#requirements)
+- [Metadata Ingestion](#metadata-ingestion)
+- [Data Profiler](/connectors/ingestion/workflows/profiler)
+- [Data Quality](/connectors/ingestion/workflows/data-quality)
+
+{% partial file="/v1.4/connectors/ingestion-modes-tiles.md" variables={yamlPath: "/connectors/database/greenplum/yaml"} /%}
+
+## Requirements
+{%inlineCallout icon="description" bold="OpenMetadata 1.4 or later" href="/deployment"%}
+To deploy OpenMetadata, check the Deployment guides.
+{%/inlineCallout%}
+
+Connector was tested on Teradata DBS version 17.20. Since there are no significant changes in metadata objects, so it should work with 15.x, 16.x versions.
```

Comment:
@gpby can we document what are the user permissions one needs to setup to get the metadata from Teradata?

Reply:
By default all Teradata users has access to metadata objects (dbc.tablesv, dbc.columnsv, etc.) because public role has access to those objects. So DBA should do some explicit revokes to limit access to metadata object (e.g. [KB0024146](https://support.teradata.com/knowledge?id=kb_article_view&sysparm_article=KB0024146))